### PR TITLE
Updated Quick start guide.

### PR DESCRIPTION
**Pull request by swatid-seagate**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

### CODE REVIEW

**Review by johnbent — CHANGES_REQUESTED** [review]

File: QUICK_START.md [hunk 1]
```diff
@@ -6,25 +6,15 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node: [LINK](doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst)
     1. On an AWS EC2 instance: [LINK](doc/integrations/AWS_EC2/README.md)
     1. Across a cluster: TODO
-1. Build entire CORTX into binaries: [LINK](doc/community-build/Building-CORTX-From-Source-for-SingleNode.md)
-    1. Run it on a single node: [LINK](doc/community-build/ProvisionReleaseBuild.md)
-    1. Run it on a AWS: [LINK](doc/community-build/cloud/AWS/README.md)
-    1. Run it on a cluster: TODO
+1. Build entire CORTX into binaries: [LINK](./doc/community-build/docker/cortx-all/README.md)
+    1. Run it on a single node: [LINK](https://github.com/Seagate/cortx-k8s/blob/stable/doc/cortx-aws-k8s-installation.md)
+    1. Run it on a AWS: [LINK](https://github.com/Seagate/cortx-k8s/blob/stable/doc/cortx-aws-k8s-installation.md)
```
[the remark this answers is not in the document]
Comment (swatid-seagate) [reply QUICK_START.md]: Modified the file according to your suggestion and updated the PR.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: QUICK_START.md [hunk 2]
```diff
@@ -6,25 +6,16 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node: [LINK](doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst)
     1. On an AWS EC2 instance: [LINK](doc/integrations/AWS_EC2/README.md)
     1. Across a cluster: TODO
-1. Build entire CORTX into binaries: [LINK](doc/community-build/Building-CORTX-From-Source-for-SingleNode.md)
-    1. Run it on a single node: [LINK](doc/community-build/ProvisionReleaseBuild.md)
-    1. Run it on a AWS: [LINK](doc/community-build/cloud/AWS/README.md)
-    1. Run it on a cluster: TODO
+1. Run it on AWS in Kubernetes: [LINK](https://github.com/Seagate/cortx-k8s/blob/stable/doc/cortx-aws-k8s-installation.md)
```
Comment (johnbent) [inline QUICK_START.md:9]: instead of 'in Kubernetes' it should say 'using Kubernetes'

File: QUICK_START.md [hunk 3]
```diff
@@ -6,25 +6,16 @@ This document provides quick access to several different ways to set up CORTX fo
...
+2. Build entire CORTX into binaries: [LINK](./doc/community-build/docker/cortx-all/README.md)
```
Comment (johnbent) [inline QUICK_START.md:10]: This shouldn't be a 2. All numbers should be 1 and the github server will render them automatically when they serve the page.

File: QUICK_START.md [hunk 4]
```diff
@@ -6,25 +6,16 @@ This document provides quick access to several different ways to set up CORTX fo
...
+    1. Run it on a single node: [LINK](https://github.com/Seagate/cortx-k8s/blob/stable/doc/cortx-aws-k8s-installation.md)
+    
+    
 1. Build just the block storage layer (motr) from the source: [LINK](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst)
     1. Run it on a single node: [LINK](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst)
     1. Run it across a cluster: [LINK](https://github.com/Seagate/cortx-motr/wiki/Build-Motr-from-Source-in-a-Cluster)
 1. Use a pre-built virtual machine to quickly test the block storage layer (motr)
     1. Run it on a single node: [LINK](https://github.com/Seagate/cortx-motr/releases/tag/ova-centos79)
-3. Build just the datapath (motr + S3) from the source: [LINK](https://github.com/Seagate/cortx-s3server/blob/main/docs/CORTX-S3%20Server%20Quick%20Start%20Guide.md)
```
Comment (johnbent) [inline QUICK_START.md:18]: This shouldn't be a 3, should be a 1

Comment (johnbent) [reply QUICK_START.md:18]: Oh, I see you are removing this so the numbering here doesn't matter.

**Review by johnbent — CHANGES_REQUESTED** [review]

**Pull request closed without merging**